manifests/autotune-operator-rolebinding.yaml_template

```diff
@@ -10,3 +10,16 @@ subjects:
 - kind: ServiceAccount
   name: autotune-sa
   namespace: "{{ AUTOTUNE_NAMESPACE }}"
+---
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRoleBinding
+metadata:
+ name: autotune-admin-crb
+roleRef:
+ apiGroup: rbac.authorization.k8s.io
+ kind: ClusterRole
+ name: cluster-admin
```

Comment:
Do we really need a cluster-admin role?  We need to document the exact uses and the need for this as we are then asking autotune to be a superuser